docs/ref/utils.txt

```diff
@@ -65,7 +65,13 @@ need to distinguish caches by the ``Accept-language`` header.
 
 .. function:: add_never_cache_headers(response)
 
-    Adds headers to a response to indicate that a page should never be cached.
+    Adds "Cache-Control: max-age=0, no-cache, no-store, must-revalidate" header
+    to a response to indicate that a page should never be cached.
+
+.. versionchanged:: 1.9
```

Comment:
This annotation and its contents should be indented another level like the others on this page.